policies/transparency_logs.rego

```diff
@@ -0,0 +1,130 @@
+package hacbs.contract.transparency_urls
+
+import future.keywords.every
+
+#
+# Sanity check the configuration of tekton-chains in the cluster
+#
+# Actually this is a weak test since the configuration might have changed
+# since the pipeline ran.
+#
+# Todo:
+# - This doesn't fail if the key is entirely absent, which is bad.
+# - Maybe check that rest the chains configuration is what we require.
+#
+deny[{"msg": msg}] {
```

Comment:
I think this should be in a `config` policy, we will probably validate other settings in the future.

Reply:
You mean move it to its own separate rego file, right? Agreed.

Reply:
But it could be done a a later commit. This commit is largely just copying stuff from the original PR in the other repo.

Reply:
Yeah this PR's goal is to just have the policies available for the enterprise task.